packages/ui/react-components/src/interactive-list/interactiveList.less

```diff
@@ -60,10 +60,6 @@
     outline: none;
     cursor: pointer;
 
-    &:focus-visible {
-        outline: #0067c5 auto 1px;
```

Comment:
Denne brukes for å vise fokus ved tastaturnavigering, er det erstattet med noe som er tilstrekkelig?